src/schema/v2/object_identification.ts

```diff
@@ -141,13 +158,22 @@ const NodeField: GraphQLFieldConfig<any, ResolverContext> = {
   },
   // Re-uses (slightly abuses) the existing GraphQL `resolve` function.
   resolve: (_root, { id: globalID }, context, rootValue) => {
-    const { type: typeName, id } = fromGlobalId(globalID)
+    const { id } = fromGlobalId(globalID)
+    let { type: typeName } = fromGlobalId(globalID)
```

Comment:
`typeName` here is `ArtworksRailHomeViewSection`. But the actual resolver is in `Section` [here](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/homeView/index.ts#L52). That's why I introduced this little remap object above, to point to the correct resolver object

Reply:
good catch